packages/components/src/components/expansion-panel/expansion-panel.tsx

```diff
@@ -1,11 +1,16 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
-import ArrayRenderer from './array-renderer.jsx';
+import ArrayRenderer from './array-renderer';
 import Icon from '../icon';
+import { TItem } from '../types/common.types';
 
-const ExpansionPanel = ({ message, onResize }) => {
-    const [open_ids, setOpenIds] = React.useState([]);
+type TExpansionPanel = {
+    message: { header: React.ReactNode; content: Array<React.ReactNode> & Array<TItem> };
```

Comment:
```suggestion
    message: { header: React.ReactNode; content: Array<React.ReactNode | TItem> };
```

Reply:
That was my first thought. But I get the below error.
<img width="813" alt="Screen Shot 2022-11-14 at 8 25 28 AM" src="https://user-images.githubusercontent.com/98398322/201575177-9ce00a65-8301-48c0-9a1b-05aa519d80b1.png">

Reply:
oh, ok